src/components/Table/Table.module.scss

```diff
@@ -1,5 +1,4 @@
 .container {
-  display: inline-block;
```

Comment:
tables should be block elements. the parent should be responsible for setting it's width.